### ISSUE TITLE
Productivity power tool on mac for vs 2019

### ISSUE THREAD

**Issue by Alagbara** [microsoft/VS-PPT#229]
I am trying to install procudtivity power tool for vs mac 2019 but could not find any.
Could you please help?

**Comment by olegtk** [comment]
Hmm, there is no VS for Mac version (yet?). @KirillOsenkov, FYI.

**Comment by KirillOsenkov** [comment]
Yes, it currently only works on Windows. Which ones do you need on the Mac?

**Comment by winterdouglas** [comment]
I'd love to have the solution error visualizer, that's for me the most important one.
The shrink empty lines is also something i'd like to see there.

**Comment by gabrielmaldi** [comment]
We'd love to use "Remove and Sort Usings" features (for a whole solution/project, on save) on Mac 😄.
